js/ui/scheduler/workspaces/view_data_provider.js

```diff
@@ -697,7 +697,7 @@ class GroupedDataMapProvider {
             : startDate;
     }
 
-    findCellPositionInMap(groupIndex, startDate, isAllDay) {
+    findCellPositionInMap(groupIndex, startDate, isAllDay, index) {
```

Comment:
I suggest using one argument `options` instead of four arguments.
```
findCellPositionInMap(options) {
const { groupIndex, startDate, isAllDay, index } = options;
```